android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -499,6 +499,11 @@ protected void onTransportUpdate(List<TransportRecord> connectedTransports, bool
 		}
 	}
 
+    void createTouchScalar(ImageResolution resolution, DisplayMetrics displayMetrics) {
```

Comment:
This code is used in other places in this class, please refactor to not have duplicate code

Reply:
DONE